torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -406,7 +448,12 @@ class CppMicroGemmFP32Vec(CppMicroGemm):
     auto storec = [&](auto i) {
         constexpr int row = i / COLS;
         constexpr int col = i % COLS;
-        vc[i].store(C + row * ldc + col * VLEN);
+        {%- if output_dtype in [torch.bfloat16, torch.float16] %}
+        auto vc_lower_precision = at::vec::convert<{{input_t}}>(vc[i]);
+        vc_lower_precision.store(C + row * ldc + col * VLEN, VLEN);
```

Comment:
ditto

Reply:
Will remove it after making micro-kernel output FP32. Thanks!